projects/plugins/jetpack/_inc/client/components/jetpack-dialogue-modern/index.jsx

```diff
@@ -7,7 +7,8 @@ import onKeyDownCallback from 'utils/onkeydown-callback';
 
 class ModernOverlay extends Component {
 	maybeDismiss = e => {
-		if ( this.props.showDismiss && ( ! e.keyCode || e.keyCode === 27 ) ) {
+		// "Esc" is an IE/Edge specific value
```

Comment:
Our browserslist config indicates that we don't support IE anymore, and only Edge >= 102. Is "Esc" still relevant?

https://caniuse.com/keyboardevent-key lists Edge as having non-standard identifiers up to version 18, and full support after that.